pkg/cloud/openstack/machine/actuator.go

```diff
@@ -54,6 +54,10 @@ const (
 	RetryIntervalInstanceStatus = 10 * time.Second
 
 	TokenTTL = 60 * time.Minute
+
+	MachineCreating string = "Creating"
+	MachineRunning  string = "Running"
+	MachineFailed   string = "Failed"
```

Comment:
Is machine phase name common? Creating/Running/Faild are not in vm_state. If we follow OpenStack vm_state, The value would be Building/Active/Error.
https://developer.openstack.org/api-ref/compute/?expanded=create-server-detail,list-servers-detail